pkg/apis/istio/v1alpha1/values_types.proto

```diff
@@ -1080,10 +1077,19 @@ message PilotPolicyConfig {
   google.protobuf.BoolValue enabled = 1;
 }
 
-// Controls whether telemetry is exported for Pilot.
-message PilotTelemetryConfig {
+// Controls telemetry configuration
+message TelemetryConfig {
   // Controls whether telemetry is exported for Pilot.
   google.protobuf.BoolValue enabled = 1;
+
+  // Use telemetry v2.
+  PilotTelemetryV1Config v2 = 3;
+}
+
+// Controls whether pilot will configure telemetry v1.
+message PilotTelemetryV1Config {
```

Comment:
This should be TelemetryV2Config?

Reply:
Yes, and blocked by https://github.com/istio/installer/pull/551. I can update this once that merges